apps/api-journeys/prisma/seed.ts

```diff
@@ -47,19 +47,45 @@ async function main(): Promise<void> {
       parentOrder: 0
     }
   })
-  await prisma.block.create({
+  const video = await prisma.block.create({
     data: {
       journeyId: journey.id,
       blockType: 'VideoBlock',
       parentBlockId: card.id,
       extraAttrs: {
-        src: 'https://playertest.longtailvideo.com/adaptive/elephants_dream_v4/index.m3u8',
+        video: {
```

Comment:
```suggestion
        content: {
```

maybe content would be more appropriate here. Otherwise its a video prop inside a video.